### DISCUSSION TITLE
Nice image viewer but lack of features

### DISCUSSION THREAD

**Discussion by fairking** [d2phap/ImageGlass#1642] [General] [answered]
Hi guys,

Thanks for the nice app. I have just downloaded and looked at that.

The only things I am missing is Print button and Zoom in/out.

If it would be there I would give 5 starts.

Regards

**Comment by d2phap** [comment, marked as answer]
You can add those buttons from Settings > Toolbar:

![image](https://github.com/d2phap/ImageGlass/assets/3154213/aaf2e507-bc43-4e54-8643-a2d7f4d4b688)

Or find it from main menu > File > Print, or Zoom > Zoom in / Zoom out

**Comment by fairking** [comment]
Thanks.  👍
